portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -308,110 +308,128 @@ definition {
+			ContentDashboard.editWebContent(webContentTitle = "WC WebContent New Title");
+		}
+		
+		task ("Check if content is correct") {
+			AssertTextPresent(
+				locator1 = "ContentDashboard#FIRST_ELEMENT_LISTED_MODIFIED_DATE",
+				value1 = "0 Seconds Ago");
+		}
 
-		WebContent.editCP(
-			webContentContentEdit = "WC WebContent New Content",
-			webContentTitle = "WC WebContent Title",
-			webContentTitleEdit = "WC WebContent New Title");
+		task ("Delete WC") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Content",
-			panel = "Applications",
-			portlet = "Content Dashboard");
+			WebContent.viewTitle(webContentTitle = "WC WebContent New Title");
 
-		AssertTextPresent(
-			locator1 = "ContentDashboard#FIRST_ELEMENT_LISTED",
-			value1 = "WC WebContent New Title");
+			WebContent.deleteCP(
+				siteURLKey = "test-site-name",
+				webContentTitle = "WC WebContent New Title");
+		}
 	}
 
 	@description = "Validade if a Web Content not associated with a Display Page Template is shown"
 	@priority = "5"
-	test ContentDashboardExhibitWebContent {
-		JSONLayoutpagetemplate.addDisplayPageTemplateEntry(
-			contentType = "Web Content Article",
-			displayPageTemplateEntryName = "Display Page Name",
-			groupName = "Test Site Name",
-			subType = "Basic Web Content");
+	test ShouldExhibitWebContentNotAssociatedWithDisplayPageTemplate {		
+		task ("Create WC via JSON"){
+			JSONWebcontent.addWebContent(
+				content = "WC Content",
+				groupName = "Test Site Name",
+				title = "WC Title");
+		}
 
-		JSONWebcontent.addWebContent(
-			content = "WC Content",
-			groupName = "Test Site Name",
-			title = "WC Title");
+		task ("Check if WC is not associated in Content Dashboard") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Content",
-			panel = "Applications",
-			portlet = "Content Dashboard");
+			AssertElementNotPresent(locator1 = "//span[contains(@title, 'This content has a display page')]");
+		}
 
-		AssertTextPresent(
-			locator1 = "ContentDashboard#FIRST_ELEMENT_LISTED",
-			value1 = "WC Title");
+		task ("Delete WC") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			WebContent.viewTitle(webContentTitle = "WC Title");
+
+			WebContent.deleteCP(
+				siteURLKey = "test-site-name",
+				webContentTitle = "WC Title");
+		}
 	}
 
 	@description = "Validade if a Web Content associated with a Display Page Template is shown"
 	@priority = "5"
-	test ContentDashboardExhibitWebContentAssociatedDisplayPageTemplate {
-		JSONWebcontent.addWebContent(
-			content = "WC WebContent Content",
-			groupName = "Test Site Name",
-			title = "WebContent Title");
+	test ShouldExhibitWebContentAssociatedWithDisplayPage {
+		task ("Add a site via JSON") {
+			JSONGroup.addGroup(groupName = "Test Site Name");
+		}
```

Comment:
Why you create another Test Site Name? You are already creating it in the setUp

Reply:
Totally, got confused form some reason. I removed this now.